packages/dashmate/configs/system/base.js

```diff
@@ -33,6 +33,7 @@ const { version } = require('../../package.json');
 
 const prereleaseTag = semver.prerelease(version) === null ? '' : `-${semver.prerelease(version)[0]}`;
 const dockerImageVersion = `${semver.major(version)}.${semver.minor(version)}${prereleaseTag}`;
+const envoyImageVersion = `${semver.major(version)}`;
```

Comment:
Shouldn't we just use envoy base image version?

Reply:
Sure, I've done a release to match the latest 1.22 branch from upstream, and pinned base config to `dashpay/envoy:1.22.11`